### DISCUSSION TITLE
multi-level mosaicJSON

### DISCUSSION THREAD

**Discussion by vincentsarago** [developmentseed/mosaicjson-spec#22] [Ideas]
extension to create multiple zoom range mosaicJSON

```
{
  "bounds": [ 34.8046875, -20.3034175184893, 35.5078125, -19.31114335506464 ],
  "center": [ 35.15625, -19.80728043677697, 10 ],
  {minzoom: 0, maxzoom: 9, : ["links to mosaicJSON"]},
  {minzoom: 10, maxzoom: 14, : ["links to mosaicJSON"]},
  {minzoom: 15, maxzoom: 18, : ["links to mosaicJSON"]},
}
```

**Comment by sharkinsspatial** [comment]
👍 I literally typed up this exact issue before looking and then saw this when I went to post it.  😄

**Comment by kylebarron** [comment]
Do you enforce that no `minzoom`-`maxzoom` pairs overlap? It seems like it would be tricky to try to combine overlapping mosaic JSONs at an intermediate zoom.

Right now my workaround for multiple zoom ranges in a mosaicJSON is to have separate Mapbox GL layers that each point to an individual mosaicJSON. One downside of that is that Mapbox only overzooms within a layer. So in [my project](https://kylebarron.dev/serverless-aerial-imagery), if you start at zoom 7, initial Landsat tiles are loaded, and those tiles are overzoomed at layers 7-12 while higher resolution tiles are generated at the higher zooms. But once you cross zoom 12, everything is blank while the NAIP tiles are generated, since it's a separate Mapbox layer.
